### ISSUE TITLE
Is that possible to inference more than 4 lanes with the CULane dataset?

### ISSUE THREAD

**Issue by passion3394** [XingangPan/SCNN#57]
is that possible to inference more than 4 lanes with the CULane dataset?
My task needs to detect more than 4 lanes, what should I do if I want to do this task? thanks!!!

**Comment by cardwing** [comment]
@passion3394 , please refer to [How to train SCNN on my own dataset #7](https://github.com/XingangPan/SCNN/issues/7)

**Comment by passion3394** [comment]
@cardwing yes, I have seen the #7 issue, I will try it. Have you tried to inference more than 4 lanes? and what's the precision ?

**Comment by cardwing** [comment]
@passion3394 , the performance depends on the proportion of the number of lanes and the difficulty of road scenarios. I have not tested models predicting more than 4 lanes.

**Comment by cardwing** [comment]
@passion3394 , you can also refer to [this repo](https://github.com/cardwing/Codes-for-Lane-Detection).

**Comment by passion3394** [comment]
@cardwing ok, thanks for your work.

**Comment by cardwing** [comment]
@passion3394 , you are welcome.
